docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+
+## Get the Source Code
+
+Clone the device-usb-camera repository:
+
+   ```bash
+   git clone https://github.com/edgexfoundry/device-usb-camera.git
+   ```
+
+## Configuration options
+### Configurable RTSP server hostname and port
+The hostname and port of the RTSP server can be configured in the `[Driver]` section of the [configuration.toml](../cmd/res/configuration.toml) The default vaules can be used in this guide.
+
+For example:
+```yaml
+[Driver]
+  RtspServerHostName = "localhost"
+  RtspTcpPort = "8554"
+```
+<p align="left">
+      <i>Sample: Snippet from configuration.toml</i>
+</p>
+
+## Deploy EdgeX and USB Device Camera Microservice
+### Building the docker image
+```shell
+make docker
+```
+
+### Run the Service
+
+1. Navigate to the `edgex-compose/compose-builder` directory.
+
+2. Run EdgeX with the microservice:
+
+   ```bash
+    make run no-secty ds-usb-camera 
+   ```
+
+## Verify Service and Device Profiles
+
+1. Check the status of the container:
+
+   ```bash 
+   docker ps
+   ```
+
+   The status column will indicate if the container is running and how long it has been up.
+
+   Example Output:
+
+   ```docker
+   CONTAINER ID   IMAGE                                         COMMAND                  CREATED       STATUS          PORTS                                                                                         NAMES
+   f0a1c646f324   edgexfoundry/device-usb-camera:0.0.0-dev                        "/docker-entrypoint.…"   26 hours ago   Up 20 hours   127.0.0.1:8554->8554/tcp, 127.0.0.1:59983->59983/tcp                         edgex-device-usb-camera                                                                   edgex-device-onvif-camera
+   ```
+
+2. Check that the device service is added to EdgeX:
+
+   ```bash
+   curl -s http://localhost:59881/api/v2/deviceservice/name/device-usb-camera | jq
+   ```
+   Successful:
+   ```json
+   {
+      "apiVersion": "v2",
+      "statusCode": 200,
+      "service": {
+         "created": 1658769423192,
+         "modified": 1658872893286,
+         "id": "04470def-7b5b-4362-9958-bc5ff9f54f1e",
+         "name": "device-usb-camera",
+         "baseAddress": "http://edgex-device-usb-camera:59983",
+         "adminState": "UNLOCKED"
+      }
+   }
+   ```
+   Unsuccessful:
+   ```json
+   {
+      "apiVersion": "v2",
+      "message": "fail to query device service by name device-usb-camera",
+      "statusCode": 404
+   }
+   ```
+## Adding Devices using REST API
+Devices can either be added to the service by defining them in a static configuration file, discovering devices dynamically, or with the REST API. For this example, the device will be added using the REST API.
+
+1. Edit the information to appropriately match the camera. The device's protocol properties contain:  
+   * **Path** is a file descriptor of camera created by the OS. You can find the path of the connected USB camera through [v4l2-ctl](https://linuxtv.org/wiki/index.php/V4l-utils) utility.
```

Comment:
What command do they have to specify to get the video path? Going to the link goes to a generic page that talks about the utility, but does not explicitly say how to use the utility to get the path. Maybe some blub about how to install in Ubuntu 20.04 and execute the command to show /dev/video* devices.